update.php

```diff
@@ -508,3 +508,13 @@ function update_1351()
 
 	return Update::SUCCESS;
 }
+
+function update_1354()
```

Comment:
Instead of implementing some update function that shifts the content, I suggest some field renaming in a `pre_update_1354` function. That function is called before the structure changes are done. So you can just rename the field.

So no orphaned field is left out there.

Reply:
Done.